Extension/lib/prefs.js

```diff
@@ -48,7 +48,11 @@ var Prefs = (function () {
             }
             return browser;
         })(),
-        betaYandex: navigator.userAgent.indexOf("Yowser") >= 0
+        betaYandex: navigator.userAgent.indexOf("Yowser") >= 0,
```

Comment:
I mean that we have Prefs.browser field, which already detects yandex browser. Why more fields?